test.sh

```diff
@@ -18,6 +18,7 @@ function usage()
     echo "Usage: $0 [-m MARKEXPR] [-k EXPRESSION] [-p PATH] [-s] all|<framework-name>"
     echo "-m passed to pytest directly [default -m \"sanity and not azure\"]"
     echo "-k passed to pytest directly [default NONE]"
+    echo "-f module to test passed directly to pytest [default test_*.py]"
```

Comment:
Yup, just tried that locally and it's fine, not sure why it didn't work earlier when I tested it (probably didn't do it right). Anyhow, thanks for pointing it out - removed.